logging_http_client/logging_http_client_config_globals.py

```diff
@@ -122,3 +122,18 @@ def set_request_body_logging_enabled(value: bool):
 def set_response_body_logging_enabled(value: bool):
     global _response_body_logging_enabled
     _response_body_logging_enabled = value
+
+
+# Request/Response Logging Toggle =============================================
+
+_logging_level: int = 20  # 20 is the default logging level for logging.INFO
```

Comment:
**suggestion(non-blocking):** Likewise, if you applied the above refactor, this can change to default value of: `LogLevel.INFO`